components/stream-provider/README.md

```diff
@@ -0,0 +1,94 @@
+# browserd/stream-provider
+
+Represents the component that sends content from the cloud 🤕☁✨
+
+[![Build Status](https://dev.azure.com/bengreenier/browserd/_apis/build/status/stream-provider?branchName=master)](https://dev.azure.com/bengreenier/browserd/_build/latest?definitionId=11&branchName=master)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bengreenier_browserd&metric=alert_status)](https://sonarcloud.io/dashboard?id=bengreenier_browserd)
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
+
+This component represents the electron app running in the cloud that streams to our [stream-consumer](https://github.com/bengreenier/browserd/tree/repo-restructure/components/stream-consumer). It will be able to send a UI to and recieve input from the stream-consumer web app running on a browser.
```

Comment:
tree is set to your branch, should be `master` at minimum, or relative path preferably

Reply:
Unsure of how to proceed on this? How does a relative path work for a URL? Or can I just write ../stream-consumer?